### DISCUSSION TITLE
ctrl eg can't used

### DISCUSSION THREAD

**Discussion by WarlockFish** [wezterm/wezterm#4461] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### WezTerm version

20231004-113117-11dec45f x86_64-unknown-linux-gnu

### Ask your question!

When I use SSH to connect to other servers, sometimes I cannot use the ctrl shortcut key after disconnecting SSH. I entering ctrl+c,  and output; 5; 99~

```
# h @ Linx in ~ [11:05:37] [⚡255]  ⏱ 3s
✗ ;5;99~;5;58~
```

how to reset the terminal

**Comment by bew** [comment, marked as answer]
Try the `reset` command

**Reply by WarlockFish** [reply]
thank you. `reset` is ok
